### DISCUSSION TITLE
Wiring diagram for 5 pin MIDI

### DISCUSSION THREAD

**Discussion by Solone** [tttapa/Control-Surface#1035] [Q&A]
What's the standard wiring wiring configuration for the 5 pin midi in/out when using hardware serial midi and not USB? I know both pin 2 and 4 need a 220 ohm resistor?? Anybody have a wiring diagram please? Thanks

**Comment by tttapa** [comment]
The definitive answer is in the official MIDI spec on midi.org: https://midi.org/5-pin-din-electrical-specs
